Mathlib/Data/Option/Basic.lean

```diff
@@ -177,8 +177,6 @@ theorem join_pmap_eq_pmap_join {f : ∀ a, p a → β} {x : Option (Option α)}
     (pmap (pmap f) x H).join = pmap f x.join fun a h ↦ H (some a) (mem_of_mem_join h) _ rfl := by
   rcases x with (_ | _ | x) <;> simp
 
-/-- `simp`-normal form of `join_pmap_eq_pmap_join` -/
-@[simp]
```

Comment:
The left hand side is no longer in simp normal form, I think because of a new congruence lemma for `pmap`. This lemma isn't needed anywhere in Mathlib for now, so I think it's okay to just remove `@[simp]` and wait for further improvements to simp normal form for Option as it is worked on upstream.